components/system/components/Tag.js

```diff
@@ -0,0 +1,581 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/common/svg";
+
+import { LoaderSpinner } from "~/components/system/components/Loaders";
+import { css } from "@emotion/react";
+
+const STYLES_TAG_CONTAINER = css`
+  width: 100%;
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const STYLES_INPUT_CONTAINER = css`
+  width: 100%;
+  position: relative;
+`;
+
+const STYLES_DROPDOWN = css`
```

Comment:
Mind rounding the corners on the dropdown to match the rest of the UI elements (using border radius)? It looks a little off when it's square corners when everything else is rounded.